src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueClosureTests.cs

```diff
@@ -995,6 +1210,105 @@ .maxstack  2
         }
 
         [Fact]
+        public void LocalFunctions_UpdateAfterAdd()
+        {
+            var source0 = MarkedSource(@"
+using System;
+
+class C
+{
+    static int G(Func<int, int> f) => 1;
+
+    static object F()
+    {
+        return G(null);
+    }
+}");
+            var source1 = MarkedSource(@"
+using System;
+
+class C
+{
+    static int G(Func<int, int> f) => 1;
+
+    static object F()
+    {
+        <N:0>int f(int a) => a + 1;</N:0>
+        return G(f);
```

Comment:
>G(f); [](start = 15, length = 5)

It's more interesting to test a local function that is not passed to a delegate -- the compiler treats a local function passed to a delegate more like a lambda. Let's add another test where the local function is not passed to a delegate.

Reply:
Thanks! Added

---
In reply to: [136188851](https://github.com/dotnet/roslyn/pull/21777#discussion_r136188851) [](ancestors = 136188851)